### PR TITLE
add dekstop part of website

### PR DESCRIPTION
**Pull request by RuriEnciso**
Update the desktop version to the website

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -49,121 +49,203 @@ <h2>My Recent Works</h2>
...
+            </div>
+            <div class="container">
+                <h2 class="project-tittle" >Profesional Art Printing Data</h2>
+                <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul class="project-tools">
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+            <button class="project-button" type="button">See Project</button>
```
Comment (AbayomiOlaoye) [inline index.html:126]: - [ ] Kindly check the video above to have a clearer picture of how your portfolio is rendering at the moment. The behaviour is not so smooth for the following reasons:
  - [ ] No background image is showing
  - [ ] No project description can be seen or is readable on the page
  - [ ] Projects section is not responsive
  - [ ] The hovering effects that allows some content of the project card to be hidden while the `see project` action pops up has not been implemented yet or not visible.
  - [ ] The style properties are hardcoded with absolute units (e.g. `px`) causing overflows, and some elements are too big - the project cards such that screen sizes above `960px` can not contain a `3` column grid to match design.

This is what gets rendered :point_down: 
image

File: index.html [hunk 2]
```diff
@@ -180,19 +262,21 @@ <h3 class="framework-text">Skills</h3>
                 <textarea rows="4" cols="50" maxlength="500" id="msj" placeholder="Enter text here" required></textarea>
                 <button class="form-buttom" type="submit">Get in touch</button>
```
Comment (AbayomiOlaoye) [inline index.html:264]: - [ ] It's a bit funny how your button seems to be running away from getting clicked. It keeps floating back and forth upon clicking even. Please kindly make it fixed at a spot, and work on implementing different styles depending on user's interaction. Here is a brief screencast of the button in action: 

[Screencast from 05-03-2023 00:44:32.webm](https://user-images.githubusercontent.com/99715640/222934526-4f95d8e9-e90f-4b56-90f9-8b6e1c7ca80b.webm)

File: index.html [hunk 3]
```diff
@@ -49,121 +49,203 @@ <h2>My Recent Works</h2>
...
-                <li>Codekit</li>
-                <li>Github</li>
-                <li>Codepen</li>
-                <li>Gitlab</li>
-                <li>Terminal</li>
-            </ul>
-        </section>
-        <section>
+
+        <section class="website-form">
```
Comment (AbayomiOlaoye) [inline index.html:249]: - [ ] As a whole, I believe that this screencast will guide you on how your page looks when compared with Figma's design. Kindly remember that your work should be as close as possible to the figma design. Here is the video:

[Screencast from 05-03-2023 00:39:52.webm](https://user-images.githubusercontent.com/99715640/222934697-a6d536c3-efa8-4880-8789-966fee57055c.webm)

- Here is also an attached Figma design (Annotated version):
  - [ ] Kindly note how the sections have equal widths
  - [ ] How the images are all clearly visible
  - [ ] How the sections are not collapsing with both sides of the available viewport
  - [ ] How the background images are showing well
  - [ ] The stunning use of `margin` and `padding` for spacing.

image

**Review by AbayomiOlaoye — CHANGES_REQUESTED** [review]
## STATUS: CHANGE REQUIRED ♻️ 

Hi @RuriEnciso :wave: , 

Good job with this milestone so far!
There are just a few more issues to work on before moving on to the next project but you are almost there!

so_close

## Highlights
- Good use of GitHub flow :heavy_check_mark: 
- Meaningful commit messages :heavy_check_mark: 
- No linter error :heavy_check_mark: 
- Descriptive ReadMe documentation :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Shedrack-Sunday — APPROVED** [review]
Hi @RuriEnciso 

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

image


# Highlights
- No linter errors ✔️
- Good implementations ✔️


## Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

__check the comments for details

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**